rpc.go

```diff
@@ -126,21 +126,24 @@ func (r rpc) Channel(service string) transport.Channel {
 }
 
 func (r rpc) Start() error {
-	callServe := func(i transport.Inbound) func() error {
+	startInbound := func(i transport.Inbound) func() error {
 		return func() error {
 			return i.Start(r)
 		}
 	}
 
 	var wait sync.ErrorWaiter
 	for _, i := range r.inbounds {
```

Comment:
I can switch the ordering but note that the inbounds don't actually consume the RPC object. They get the `Handler` they need to serve. The `RPC` object implements the `Handler` interface.